File: Task/Sum-of-first-n-cubes/C-sharp/sum-of-first-n-cubes.cs
using System; using static System.Console;
class Program { static void Main(string[] args) {
    for (int i=0,j=-6,k=1,c=0,s=0;s<1600000;s+=c+=k+=j+=6)
      Write("{0,-7}{1}",s, (i+=i==3?-4:1)==0?"\n":" "); } }

File: Task/Sum-of-first-n-cubes/Lua/sum-of-first-n-cubes.lua
do  -- show the sums of the cubes upto n for 0 <= n < 50
    local cubeSum = 0
    for n = 0, 49 do
        cubeSum = cubeSum + ( n * n * n )
        io.write( string.format( "%8d", cubeSum ) )
        if n % 10 == 9 then io.write( "\n" ) end
    end
end
